src/kernel-install/90-uki-copy.install

```diff
@@ -49,8 +49,8 @@ case "$COMMAND" in
 esac
 
 if ! [ -d "$UKI_DIR" ]; then
-    echo "Error: entry directory '$UKI_DIR' does not exist" >&2
-    exit 1
+    [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "creating $UKI_DIR"
```

Comment:
should still go to `>&2` no?

Reply:
Using stdout is consistent with other places doing verbose logging.